code/modules/clothing/glasses/glasses.dm

```diff
@@ -194,6 +194,15 @@
 	flags_armor_protection = 0
 	flags_equip_slot = SLOT_EYES|SLOT_FACE
 
+/obj/item/clothing/glasses/eyepatch_white
```

Comment:
make this a subtype of /obj/item/clothing/glasses/eyepatch

Reply:
Done